frontend/public/components/utils/headings.tsx

```diff
@@ -81,7 +81,7 @@ export const ResourceOverviewHeading: React.SFC<ResourceOverviewHeadingProps> =
       </Link>
     </div>
     <div className="co-actions">
-      <ActionsMenu actions={actions.map(a => a(kindObj, resource))} />
```

Comment:
this is not going to help. You need to load and pass the actual resource needed for the actions. Please see ResourceKebab - it is is not done correctly here either and should actually not include the firehose (TODO for future).

Reply:
:+1: Thanks, didn't notice that :-( fixing.